MM/5.9.0/config/media_manager_5/labels/Administration.dcl

```diff
@@ -2975,4 +2975,195 @@ resource configservice_label administration_tools_analytics_block_builder_title
   ]
 }
 
+resource configservice_label administration_tools_dashboard_tool_add_table {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_ADD_TABLE'
+  group = 'administration-tools - dashboard-tool'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Add table'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tilføj tabel'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_dashboard_tool_add_chart {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_ADD_CHART'
+  group = 'administration-tools - dashboard-tool'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Add chart'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tilføj diagram'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_analytics_table_something_went_wrong_when_fetching_the_data {
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_TABLE_SOMETHING_WENT_WRONG_WHEN_FETCHING_THE_DATA'
+  group = 'administration-tools - analytics-table'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Something went wrong when fetching the data'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Noget gik galt under indlæsningen af data'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_analytics_table_total {
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_TABLE_TOTAL'
+  group = 'administration-tools - analytics-table'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Total'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Total'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_analytics_table_builder_how_many_entries_to_show {
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_TABLE_BUILDER_HOW_MANY_ENTRIES_TO_SHOW'
+  group = 'administration-tools - analytics-table-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'How many rows to show?'
```

Comment:
Current: How many rows to show?
Alterna1: Number of rows to show
Alterna2: Amount of rows to show